.github/workflows/main.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: ⎔ Setup node
         uses: actions/setup-node@v2
         with:
-          node-version: 14
+          node-version: 16
```

Comment:
I think we should use https://github.com/actions/setup-node/blob/main/action.yml#L10-L11 to only have it one place.

Reply:
How do you do that?, you just add this instead ?
```
node-version-file: .nvmrc
```

Reply:
Yea exactly :+1: